lib/foreman_templates/engine.rb

```diff
@@ -94,6 +94,12 @@ class Engine < ::Rails::Engine
               description: N_('Custom commit message for templates export'),
               default: 'Templates export made by a Foreman user',
               full_name: N_('Commit message'))
+            setting('template_sync_http_proxy_policy',
+              type: :string,
+              description: N_('HTTP proxy policy for template sync'),
```

Comment:
> About describing the options: Ideally I want to reach state where options themselves would be descriptive enough.

:+1: :100: 

> Probably the most unclear is "Custom HTTP proxy". Do you think it would be okay to is provide more detail for that one and omitting the other two in the description?

Yes! I like this idea. How about just adding a few words to explain that if users select "Custom HTTP proxy", they will be prompted for the proxy (because after you select `Custom`, a new field appears). Together with the new field now including the hint about "Infrastructure > HTTP proxies", users shouldn't get lost -- and you avoid overdescribing the actual options. WDYT?

> I wonder if capitalizing template sync is a good idea. Is this to keep consistency with a different part of the UI?

Yes, that was the idea. But now that I'm looking again, is Template Sync capitalized only when it's used as a heading..? Okay, please disregard these comments :) We don't list is as a term to be capitalized in the docs [CONTRIBUTING.md](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md#capitalization) so let's not do it here either.

Reply:
Work for me! Now we won't have to explain all of this separately on the docs side. Thanks!